Cython/Compiler/Options.py

```diff
@@ -207,6 +207,7 @@ def get_directive_defaults():
     'old_style_globals': False,
     'np_pythran': False,
     'fast_gil': False,
+    'fastcall_args': [None, None],
```

Comment:
The list value seems to have some magic meaning that suggests a comment right here.